README.rst

```diff
@@ -18,3 +18,13 @@ Prerequisites
 * `Traits <https://pypi.python.org/pypi/traits>`_
 * `NumPy <https://pypi.python.org/pypi/numpy>`_
 * `SciPy <https://pypi.python.org/pypi/scipy>`_
+
+Development Environment Setup
+-----------------------------
+
+To setup the EDM environment for this project::
+
+    $ edm install -e name_of_your_scimath_env click, setuptools, coverage
```

Comment:
Let's use the form without the commas.

```suggestion
    $ edm install -e name_of_your_scimath_env click setuptools coverage
```

Reply:
Can you please double-check this list? We definitely need click, and we might also need `setuptools`, but I'd be surprised if we need `coverage`.

Reply:
I just checked, the coverage is indeed not necessary, removed. And it seems not need setuptools as well as tested separately... I just push a new version with format update and these two removed